recipes/tango_checkup/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "tango_checkup" %}
```

Comment:
Looking at the project repo and README, the project name is `tango-checkup`. I'd use that name for the conda package.

Both sdist and wheels archives are now normalised and `-` are converted to `_`. You can `pip install` both `tango-checkup` and `tango_checkup`, but that's not the case for conda. Package name must match exactly.

You'll have to change the recipe directory name as well.